public/js/angular/controllers.js

```diff
@@ -254,44 +246,42 @@ angular
       $scope.weblit = weblit;
 
       $scope.wlcPoints = CONFIG.wlcPoints;
-
     }
   ])
-  .controller('competencyMediaGenController', ['$scope', '$sce',
+  .controller("competencyMediaGenController", ["$scope", "$sce",
```

Comment:
it's actually a pretty widely used pattern for avoiding switches and long if cascades due to how JS works, so I'd mark it as one of those javascript-isms, worth learning about when you get into the language.